src/Core/Domain/Carrier/Command/AddCarrierCommand.php

```diff
@@ -80,4 +85,29 @@ public function getLogoPathName(): ?string
     {
         return $this->logoPathName;
     }
+
+    public function isShippingHandling(): bool
```

Comment:
As commented earlier is there a way to improve this naming? The meaning is not very clear